src/Host.h

```diff
@@ -773,6 +778,18 @@ private slots:
     bool mCompactInputLine;
 
     QTimer purgeTimer;
+
+    // How to display (most) incoming control characters in TConsoles:
+    // 0 = as is, no replacement
+    // 1 = as Unicode "Control Pictures" - use Unicode codepoints in range
+    //   U+2400 to U+2421
+    // 2 = as "OEM Font" characters (most often seen as a part of CP437
+    //   encoding), see the corresponding Wikipedia page, e.g.
+    //   EN: https://en.wikipedia.org/wiki/Code_page_437
+    //   DE: https://de.wikipedia.org/wiki/Codepage_437
+    //   RU: https://ru.wikipedia.org/wiki/CP437
+    int mControlCharacterHandlingMode;
```

Comment:
Yeah, that makes sense, only issue with `enum`s is that IIRC sometimes it can take more effort to read/write them from/to a `QDataStream` as Qt/compiler gets unhappy about converting between a `enum` and the underlying integer form. I'll give it a go and see if I can find the magic invocations to use...